Design edits

pechouc opened the pull request:
Once #19, which is dedicated to the move from Streamlit to Dash and the deployment of the app via Heroku, has passed, this PR aims at bringing several design and style corrections. The changes essentially come from Melissa's message on Slack ([link](https://pyronear.slack.com/archives/C036SUU5WGJ/p1654764308236539)). They are the following:

- [x] Adapting the font sizes that are too big, especially on mobile
- [x] "visitez notre site web" rather than "à propos" for the navigation bar link
- [x] "Envoi de photo" to be changed into "Sélectionner ou prendre une photo"
- [x] Changing the text of "Télécharger une ou plusieurs photo(s)"
- [ ] Adding a loader for the image upload
- [x] Positioning the date picker on the current date
- [x] Using a native picker for the time input
- [x] Etiquette to be reviewed
- [x] Editing the text with the condition acceptance button
- [x] Send button truncated on mobile
- [x] Changing the colour of the error messages
- [x] Changing the final success message

pechouc commented:
Here are a few comments for some of the points mentioned in the first message above.

## Adding a loader for the image upload

I didn't find any very satisfying solution in the standard Dash framework. However, the alternative component provided by `dash-uploader` comes with a built-in progress bar ([link](https://github.com/np-8/dash-uploader)). Should we explore this solution or should we stick with the standard framework in your opinion?

## Positioning the date picker on the current date

We can add the current date as a default value for the input but then, if the person forgets to update the date, it will not send an error message and we could thereby get wrong dates. I would not be in favour of such a modification. That being said, I have modified the code so that at least the current month is shown when opening the small calendar. Let me know what you think!

## Using a native picker for the time input

A bit like for `dash-uploader`, the `dash-mantine-components` comes with a cleaner time input. Here are the links [to the general documentation](https://dash-mantine-components.herokuapp.com) and specifically [to the time input](https://www.dash-mantine-components.com/components/timeinput).

frgfm commented on Dockerfile:
```diff
@@ -16,4 +16,4 @@ RUN apt-get update \
 COPY ./src src/
 
-CMD ["streamlit", "run", "src/app.py"]
+CMD ["gunicorn", "-b 0.0.0.0:80", "src.app:server"]
```
that would be better to have the default python app running command (gunicorn would be the docker-compose command: the reason behind this is that gunicorn can be replaced with uvicorn or other tools)

frgfm commented on src/app.py:
```diff
@@ -1,90 +1,980 @@
+import base64
+import csv
+import os
+from datetime import date, datetime
 
-# This program is licensed under the Apache License version 2.
-# See LICENSE or go to <https://www.apache.org/licenses/LICENSE-2.0.txt> for full license details.
```
We should keep the headers :)

frgfm reviewed (commented):
Thanks for the PR!

I think there is some overlap with the other PR, could you merge "main" into your local branch?

frgfm commented on src/app.py:
```diff
@@ -45,73 +51,93 @@ def is_int(element):
-            'text-indent': '5%',
+        text='Glisser / déposer une photo ici - ou parcourir',
+        text_completed='Vous avez choisi : ',
+        text_disabled="Le téléversement n'est plus disponible.",
```
téléversement ? what is that :sweat_smile:

frgfm commented on src/app.py:
```diff
@@ -45,73 +51,93 @@ def is_int(element):
+        pause_button=False,
+        disabled=False,
+        filetypes=['png', 'jpeg', 'jpg'],
+        max_file_size=1024,
```
is it in Mb? because if it's in Kb, I think we should increase this

frgfm commented on src/app.py:
```diff
@@ -45,73 +51,93 @@ def is_int(element):
+    #                             "text-decoration": "underline",
+    #                             'color': '#283B3D',
+    #                             'font-size': '14px',
+    #                         }
+    #                     )
+    #                 ]
+    #             ),
+
+    #             html.Div(
+    #                 'Maximum 200Mb - Formats acceptés : PNG, JPEG',
+    #                 style={
+    #                     'margin-top': '1%',
+    #                     'color': '#737373',
+    #                     'font-size': '10px',
+    #                     'font-weight': 'bold',
+    #                 }
+    #             ),
+
+    #             html.Div(
+    #                 'Placeholder',
+    #                 style={
+    #                     'font-size': '14px',
+    #                     'font-weight': 'bold',
+    #                     'color': '#DCE6EA',
+    #                 }
+    #             ),
+    #         ]
+    #     ),
+    #     style={
+    #         'width': '100%',
+    #         'borderRadius': '10px',
+    #         'textAlign': 'left',
+    #         'text-indent': '5%',
+    #         'background-color': '#DCE6EA',
+    #         'margin-top': '2%'
+    #     },
+    #     # Allow multiple files to be uploaded
+    #     multiple=False
+    # )
+
```
can be removed

frgfm reviewed (commented):
Thanks for the edits! A few commented sections can be removed

pechouc commented:
## What is done

- Went through the different tasks listed (see two remarks below)
- Merged `main` into this branch
- Added the headers
- Deployed again on Heroku

## Two remarks

1. Deploying `dash-uploader` on Heroku is apparently quite tricky and the deployment failed. I have moved back to the old upload component without the progress bar. I am not sure I see a way to inform the user about the loading of the file, except with a comment of the type "L'opération peut prendre quelques secondes" below the upload box.

2. See above regarding the date at which the date picker should open. 

## Other

@frgfm, I am not sure to understand your remark about `gunicorn` and I am ok with any set-up that seems appropriate to you. Don't hesitate to add a commit yourself if you feel like explaining the issue would be longer 😅 If you do so, note that `gunicorn` is used not only in the Dockerfile but also in the Procfile.

frgfm reviewed (approved):
Thanks!

A few things to modify in the next PRs:
- switch to lower case "SELECTIONNEZ" --> "Sélectionnez"
- add hyperlinks to the footer elements

And I'll connect the backend :+1:
